Makefile

```diff
@@ -38,6 +38,9 @@ testsuite/%: FORCE poetry-no-dev
 test: ## Run all non mgc tests
 test pytest tests: kuadrant
 
+smoke: poetry-no-dev
+	$(PYTEST) -n6 -m 'smoke' --dist loadfile --enforce $(flags) testsuite/tests
```

Comment:
`-n6` flag is not recommended due to higher probability of gateway simply not getting ready in time. I would recommend either 4 or 3 threads